CanaryCollector/Remote/Jira/JiraTicketProvider.cs

```diff
@@ -18,12 +18,18 @@ public JiraTicketProvider(Atlassian.Jira.Jira jira)
             this.jira = jira;
         }
 
-        public Task<IssueTicket[]> GetTicketsPendingMerge()
+        public async Task<IssueTicket[]> GetTicketsPendingMerge()
         {
-            var issues = jira.Issues.Queryable.Where(t => t.Status == "pending review" || t.Status == "test").Take(500);
-            return Task.FromResult(issues.Select(ReadTicket).ToArray());
+            var linearWorkflowIssues = GetLinearWorkflowIssues().Take(500).ToArray();
+            var pullRequests = GetIncompletePullRequests().Take(500).ToArray();
+            var pullRequestParents = await jira.Issues.GetIssuesAsync(pullRequests.Select(s => s.ParentIssueKey).Distinct());
+
+            return linearWorkflowIssues.Concat(pullRequestParents.Values).Select(ReadTicket).ToArray();
         }
 
+        private IQueryable<Issue> GetLinearWorkflowIssues() => jira.Issues.Queryable.Where(t => t.Status == "pending review" || t.Status == "test");

```

Comment:
The status names are `Review` and `Test` I think.

If you want include a definition (that I totally will never change):
```
public enum JiraPullRequestStatus
{
    Review,
    Approved,
    Test,
    Closed,
    Blocked,
    Reopened
}
```